common/changes/@gooddata/sdk-ui-all/release_2021-09-17-08-32.json

```diff
@@ -0,0 +1,11 @@
+{
+    "changes": [
+        {
+            "packageName": "@gooddata/sdk-ui-all",
+            "comment": "Charts and dashboard view component support data sampling through execution configuration. Note: This is only valid for Tiger backend with Vertica datasource.",
```

Comment:
Detail: Please use passive voice: Charts and dashboard view component support data sampling through execution configuration **was added**. This is how we do the other changelog entries.